src/scripts/site.js

```diff
@@ -82,18 +75,25 @@ function loadCoreDictionary() {
 }
 
 function init() {
-    ServerConnections.initApiClient();
+    serverAddress().then(server => {
+        if (!server) {
+            navigate('selectserver.html');
```

Comment:
For some reason, `window` is `undefined` in `page.js`. Probably not the best place.